src/aggregations/aggregation-builder.js

```diff
@@ -0,0 +1,64 @@
+import _ from 'lodash'
+import aggregations from './index'
+
+const AggregationBuilder = {
```

Comment:
I understand `class` is just sugar for prototypal inheritance and that composition over inheritance can be cleaner, but with what's implemented here there wouldn't be much difference either way. But this already works so instead of getting into a philosophical debate about JS inheritance I'm fine leaving this as is :)